### ISSUE TITLE
Tensorflow froze 2 variables & converted to const ops

### ISSUE THREAD

**Issue by Reavern** [tensorflow/tensorflow#13125]
Hi there, 
I'm new to Python & Tensorflow. Im using raspbian9 (stretch) & Python 2.7.13.

When i run retrain.py from the examples provided using these code:

`python retrain.py --bottleneck_dir=tf_files/bottlenecks --how_many_training_steps=50 model_dir=tf_files/inception output_graph=retrained_graph.pb output_labels=retrained_labels.txt --image_dir=img --input_binary=true`

There are no outputs for output_labels (retrained_labels.txt) & output_graph (retrained_graph.pb).
The codes below are the console outputs for the few last lines console output from the command above:

`INFO:tensorflow:2017-09-18 12:50:07.148873: Step 49: Train accuracy = 95.0%
INFO:tensorflow:2017-09-18 12:50:07.149888: Step 49: Cross entropy = 0.241774
INFO:tensorflow:2017-09-18 12:50:08.259520: Step 49: Validation accuracy = 97.0% (N=100)
INFO:tensorflow:Final test accuracy = 89.3% (N=28)
INFO:tensorflow:Froze 2 variables.
Converted 2 variables to const ops.`

Any solutions? Thanks!

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
